crates/fj-app/src/main.rs

```diff
@@ -45,6 +45,9 @@ fn main() -> anyhow::Result<()> {
 
     let args = Args::parse();
     let config = Config::load()?;
+    
+    // finds the absolute path by the directory name
+    let abs_path_for_models = PathBuf::from("./models").canonicalize().unwrap();
```

Comment:
The use of `./models` here is not correct. The `models/` directory has no special meaning for Fornjot. It just has a special meaning within this repository, because of the [configuration in `fj.toml`](https://github.com/hannobraun/Fornjot/blob/8fe463a9beba9c69c65a2ed61472a8c5db7c44c1/fj.toml#L3).

The correct path to call `canonicalize` on would be `path`, defined below, before `model` is pushed to it..

Also, this will panic, if a `models/` directory doesn't exist in the current working directory:

```
[hanno@grimbart models]$ cargo run -- -m cuboid
    Finished dev [unoptimized + debuginfo] target(s) in 0.22s
     Running `/home/hanno/Projects/main/fornjot/target/debug/fj-app -m cuboid`
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', crates/fj-app/src/main.rs:50:72
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

This renders the point of creating a better error message moot :smile: 

To get a decent error message here, we'd need to replace the `.unwrap()` with a `.with_context(|| ...)?`, and create an error message within the closure, like below.